cluster/get-kube.sh

```diff
@@ -183,6 +193,9 @@ if [[ ${KUBE_VERSION} =~ ${KUBE_CI_VERSION_REGEX} ]]; then
   # Override KUBERNETES_RELEASE_URL to point to the CI bucket;
   # this will be used by get-kube-binaries.sh.
   KUBERNETES_RELEASE_URL="${KUBERNETES_CI_RELEASE_URL}"
+elif [[ ${KUBE_VERSION} =~ ${KUBE_BAZEL_VERSION_REGEX} ]]; then
+  KUBE_VERSION=${KUBE_VERSION#bazel/}
```

Comment:
I'd tend to agree with respect to all the convoluted parsing, but someone(s) definitely went to a lot of work for all the regex validation. I like the idea of just having an escape hatch to use a specific URL without all of it though.